source/index.rst

```diff
@@ -45,22 +45,21 @@ settings follow the current recommendations.
     channel_priority_strict`` was added; see the `explanation of commands below
     <#explain-commands>`_ for details.
 
+    In August 2024, the ``defaults`` channel was removed from the recommended
+    set of channels.
```

Comment:
Someone may wish to suggest a politically appropriate explanation for why this was done.
Or we could just blandly log the change without explanation, as this PR currently does.